tests/lib/rules/require-atomic-updates.js

```diff
@@ -68,7 +129,7 @@ ruleTester.run("require-atomic-updates", rule, {
             errors: [VARIABLE_ERROR]
         },
         {
-            code: "let foo; async function x() { foo = foo + (bar ? baz : await amount); }",
+            code: "let foo; async function x() { foo = foo + (barrr ? baz : await amount); }",
```

Comment:
Not sure I understand the purpose of this change.

Reply:
Oops, this was accidental --- I temporarily changed it in order to be able to grep for a single test, but then I forgot to change it back.